stepup/tests/behat/features/selfservice.feature

```diff
@@ -33,3 +33,12 @@ Feature: A user manages his tokens in the SelfService portal
     Then I should see "The following tokens are registered for your account."
     And I should see "SMS"
     And I should see "Test a token"
+
+Scenario: A user can validate an email address when not logged into selfservice
+    Given I am logged in into the selfservice portal as "jane-a2"
+    And I register a new "Yubikey" token
+    When  I log out of the selfservice portal
```

Comment:
This is probably OK. 

I'm probably too untrusting, but have you considered adding a `Then the user is logged out` to validate the session is actually gone?

Reply:
I think enough is covered in the method itself, see \SelfServiceContext::logoutOfSelfService